performance/tests/test_eqprs.py

```diff
@@ -0,0 +1,82 @@
+from django.test import TestCase, TransactionTestCase
+from performance.admin import EQPRSFileUploadAdmin
+from django.contrib.auth.models import User
+from django.contrib.admin.sites import AdminSite
+from django.contrib import admin
+from performance.models import EQPRSFileUpload, Indicator
+from budgetportal.models.government import Department, Government, Sphere, FinancialYear
+from django.test import RequestFactory
+from django.urls import reverse
+from performance import models
+
+import performance.admin
+import os
+import time
+
+USERNAME = "testuser"
+EMAIL = "testuser@domain.com"
+PASSWORD = "12345"
+
+
+def get_mocked_request(superuser):
+    request = RequestFactory().get('/get/request')
+    request.method = 'GET'
+    request.user = superuser
+    return request
+
+
+class EQPRSFileUploadTestCase(TestCase):
+    def setUp(self):
+        self.superuser = User.objects.create_user(
+            username=USERNAME,
+            password=PASSWORD,
+            is_staff=True,
+            is_superuser=True,
+            is_active=True,
+        )
+        self.correct_csv = """ReportTitle,Textbox95,Textbox80,Textbox81,Textbox82,Textbox83
```

Comment:
please use an actual csv file for this so that it's a bit more feasible to check and maintain the test data

Reply:
done